eng/common/TestResources/New-TestResources.ps1

```diff
@@ -286,7 +286,21 @@ foreach ($templateFile in $templateFiles) {
         Log "Persist the following environment variables based on your detected shell ($shell):`n"
     }
 
-    $deploymentOutputs = @{}
+    $serviceDirectoryPrefix = $serviceName.ToUpperInvariant() + "_"
```

Comment:
I just realized you updated this directly under eng/common. That will mean this gets overwritten the next time we sync this directory. You need to make sure these changes get into eng/common in azure-sdk-tools repo before the next sync or this will likely be broken.

Reply:
Nevermind I see it has been merged now. https://github.com/Azure/azure-sdk-tools/pull/509

Reply:
This is how I've been doing it. There's no way to test it in azure-sdk-tools.

Reply:
I don't mind testing it in the language repo like this first but I think it is safer to merge the azure-sdk-tools change first and then rebase the language PR on top of the sync PR and then merge the language specific PR.

Reply:
In all the times I did it the other way, your script didn't even open a PR on azure-sdk-for-net because there was no diff. Whatever floats your boat, though. ;)

Reply:
There is a race condition. If someone else merges a eng/common change in azure-sdk-tools before your's goes in there then it might overwrite your changes under eng/common in the language repo. If everything is ordered exactly correct then there will be no issue and there won't even be a PR because there is no diff but given there are sometimes things outside of your control it is safer to merge changes into the tools repo first.